File: loopsFoundation/strongNumber.cpp
#include <iostream>
using namespace std;

int main() {
    int number, originalNumber, digit, sum = 0;

    cout << "Enter a number: ";
    cin >> number;
    originalNumber = number;
    for (int i = number; i > 0; i = i / 10) {
        digit = i % 10;

        int factorial = 1;
        for (int j = 1; j <= digit; j++) {
            factorial = factorial * j;
        }
        sum = sum + factorial;
    }

    cout<<"Sum of factorial of digits = " << sum << endl;

    if (sum == originalNumber) {
        cout << originalNumber << " is a Strong Number." << endl;
    } else {
        cout << originalNumber << " is NOT a Strong Number." << endl;
    }

    return 0;
}

File: loops/strongNumber.py
number=int(input("enter your number :"))

original=number

sum=0
while number> 0:
    digit=number%10
    fact=1
    i=1
    while i<= digit:
        
        fact=fact *i
        i=i+1
    sum=sum+fact
    number=number//10

if sum==original:
    print("strong number ")

else :
    print("not")



